sentinel-adapter/sentinel-apache-httpclient-adapter/README.md

```diff
@@ -0,0 +1,64 @@
+# Sentinel Apache Httpclient Adapter
+
+## Introduction
+
+Sentinel provides integration for OkHttp client to enable flow control for web requests.
+
+Add the following dependency in `pom.xml` (if you are using Maven):
+
+```xml
+<dependency>
+    <groupId>com.alibaba.csp</groupId>
+    <artifactId>sentinel-apache-httpclient-adapter</artifactId>
+    <version>x.y.z</version>
+</dependency>
+```
+
+We can use the `SentinelApacheHttpClientBuilder` when `CloseableHttpClient` at initialization, for example:
```

Comment:
It may be better if we pointed out that `SentinelApacheHttpClientBuilder` was a fully functioned  client builder, 
or we can produce a best practice example. (timeouts, configuration, etc)

Reply:
fixed